### PR TITLE
Make event name helper and use it on tooltip & popover to reduce dist sizes

### PR DESCRIPTION
**Pull request by GeoSot**

### CODE REVIEW

**Comment by alecpl** [comment]
Two ideas come to mind. 1) Use `eventName()` in more components. 2) Get rid of `EVENT_KEY`.

File: js/src/tooltip.js [hunk 1]
```diff
@@ -301,7 +286,7 @@ class Tooltip extends BaseComponent {
       }
 
       this._element.removeAttribute('aria-describedby')
-      EventHandler.trigger(this._element, this.constructor.Event.HIDDEN)
+      EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN))
```
Comment (alecpl) [inline js/src/tooltip.js:289]: I don't know, but can't this be `this.eventName` instead of `this.constructor.eventName` everywhere?

Comment (GeoSot) [reply js/src/tooltip.js:289]: We need to call it statically so we can use it outside the class (at least to what I have seen till now)

**Comment by GeoSot** [comment]
> Two ideas come to mind. 1) Use eventName() in more components. 2) Get rid of EVENT_KEY.

Totally aligned with your idea. I want to take it bit by bit to keep PR with few changes. In case it merged, it will be easier to continue change other components

**Review by XhmikosR — APPROVED** [review]

**Pull request merged**